### PR TITLE
Script convert .csv file to a markdown table in a .md file

### LINKED ISSUE #71: Python script to convert csv file to a table in markdown.
### I'm submitting a
- [x] feature request.

### Do you want to work on this issue?
Yes I want to work on this for SWOC.
It will take a csv file as an input and convert it into a markdown table in a .md file

### PR DESCRIPTION
**Pull request by Rahul555-droid**
Fixes #71 

### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Change logs:
Created the Script convert .csv file to a markdown table in a .md file .
added script,README and output images to the directory .

## Disclaimer
I have added the output images URL with respect to your main branch @Tejas1510 not mine so my readme file won't show it but once you merge it to the repo it will.

### CODE REVIEW

File: Python/convert_csv_to_markdown_table/csv_to_md.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+"""
+A markdown table looks like
+
+|Syntax| Description |
+|--- | ----------- |
+| Header | Title |
+| Paragraph | Text |
+
+This script takes path of csv file as an input and converts it into a markdown file
+"""
```
Comment (atarax665) [inline Python/convert_csv_to_markdown_table/csv_to_md.py:10]: Add a line break after the docstring.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Format your code according to PEP 8 Guidelines. You may use http://pep8online.com/ for this purpose.

**Comment by Rahul555-droid** [comment]
ohh forgot to use auto pep will have it done right away

**Comment by tripathyprateek** [comment]
Looks good to me. @atarax665 review and merge please 🎈

**Pull request merged**